packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -112,6 +112,23 @@ export const UIModeView: React.FC<{}> = ({
     });
   }, []);
 
+  React.useEffect(() => {
```

Comment:
This isn't necessary for our normal scenarios with Chromium. Is it worth including?